src/arrayReverse.test.js

```diff
@@ -8,13 +8,33 @@ describe(`Function 'arrayReverse':`, () => {
   });
 
   it(`should return an array`, () => {
+    const result = arrayReverse([]);
 
+    expect(Array.isArray(result)).toBe(true);
+  });
+
+  it(`should return an array with each string reversed`, () => {
+    expect(arrayReverse(['Mate', 'Academy'])).toEqual(['ymed', 'acAetaM']);
   });
 
   it(`should return an empty string
     if original array consists of an empty string`, () => {
-    expect(arrayReverse(['Mate', 'Academy'])).toEqual(['ymed', 'acAetaM']);
+    expect(arrayReverse(['', ''])).toEqual(['', '']);
   });
 
-  // write more tests here
+  it(`should return an array with each string reversed
+    including digits`, () => {
+    expect(arrayReverse(['Hell0'])).toEqual(['0lleH']);
+  });
+
+  it(`should return an array with each string reversed
+    including special characters`, () => {
+    expect(arrayReverse(['Hell@'])).toEqual(['@lleH']);
+  });
+
+  it(`should return an array with each string reversed
+    which contains more than 2 elements`, () => {
+    expect(arrayReverse(['I', 'am', 'a', 'student!']))
+      .toEqual(['!', 'tn', 'e', 'dutsamaI']);
```

Comment:
The expected result should reflect both the reversal of strings and their order. The current expectation only reverses the strings but not their order.